src/core.js

```diff
@@ -309,6 +309,21 @@ $.extend( $.validator, {
 				this.element( element.parentNode );
 			}
 		},
+		onchange: function( element, event ) {
+
+			// If target element is a select, trigger focusout on change
+			if ( element.tagName.toLowerCase() === "select" ) {
+				var _this = element;
+
+				if ( !_this.form && _this.hasAttribute( "contenteditable" ) ) {
+					_this.form = $( _this ).closest( "form" )[ 0 ];
+				}
+
+				var validator = $.data( _this.form, "validator" ),
+					settings = validator.settings;
+				settings.onkeyup.call( validator, _this, event );
```

Comment:
Sure, I'll look into working that in tomorrow, though if I remember correctly, I did it that way because the context wasn't right.

Are you referring to ```_this.hasAttribute( "contenteditable" )```? If so, I was mostly moving the same logic from the original event into this block so I didn't miss anything.